server/svix-server/src/v1/utils/mod.rs

```diff
@@ -179,10 +179,17 @@ pub fn apply_pagination_desc<
     }
 }
 
+/// Marker trait for any type that is used for iterating through results
+/// in the public API.
+pub trait IdIterator: Validate + Into<sea_orm::Value> {}
```

Comment:
I know you copied this from what was here before, but does this actually need to implement `Validate`?

Reply:
It does insofar as `ReversibleIterator` does. Because this is used as the type param to `ReversibleIterator` in `apply_pagination`. So if we're ok dropping it from that, then it can be dropped here.

Reply:
Hmmm that's annoying.